shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -2548,3 +2548,184 @@ resolveClauses
 resolveClause
     : LP_ matchString DOT_? (schemaName | MINUS_) RP_
     ;
+
+alterAuditPolicy
+    : ALTER AUDIT POLICY policyName
+        (ADD (privilegeAuditClause)? (actionAuditClause)? (roleAuditClause)? (ONLY TOPLEVEL)?)?
+        (DROP (privilegeAuditClause)? (actionAuditClause)? (roleAuditClause)? (ONLY TOPLEVEL)?)?
+        (CONDITION ( DROP | condition  EVALUATE PER (STATEMENT  | SESSION | INSTANCE)))?
+    ;
+
+privilegeAuditClause
+    : PRIVILEGES systemPrivilegeClause (COMMA_ systemPrivilegeClause)*
+    ;
+
+actionAuditClause
+    : (standardActions | componentActions)*
+    ;
+
+standardActions
+    : ACTIONS (objectAction ON (DIRECTORY directoryName | MINING MODEL objectName | objectName) | systemAction)
```

Comment:
Can you extract these rules to new sub syntax rules?

Reply:
ok